stack.yaml

```diff
@@ -39,7 +39,7 @@ packages:
   extra-dep: true
 - location:
     git: https://github.com/serokell/time-warp-nt.git
-    commit: 9ef2a546c3e3c90ea84e81892f00c5a93271ffb1
```

Comment:
This change has to be approved by @georgeee. I.e. this PR is blocked by https://github.com/input-output-hk/cardano-sl/pull/257 (which does the same change).

Since Travis checks pass, you can either remove this change and I will push your PR to master, or you can keep it and wait for https://github.com/input-output-hk/cardano-sl/pull/257 to get merged before merging this PR.